code/game/gamemodes/objective.dm

```diff
@@ -213,6 +213,12 @@ GLOBAL_LIST_INIT(potential_theft_objectives, (subtypesof(/datum/theft_objective)
 	name = "Debrain"
 	martyr_compatible = 0
 
+/datum/objective/debrain/is_invalid_target(datum/mind/possible_target)
+	if(..() || !possible_target.current.client)
+		return TRUE
```

Comment:
returning `TRUE` here is the same as returning `TARGET_INVALID_IS_OWNER`

Reply:
(on seperate lines, github merged it into one)

Reply:
That'd call the parent twice.

Reply:
You'd want to do it via `. = ..()` then `if(.)` and just a plain return (which is the same as `return .`

Reply:
Done